docs/commands/index.md

```diff
@@ -2,6 +2,8 @@
 
 To view a list of all available commands, use the sidebar. The basic commands to get started are listed below.
 
+.fmbot supports identifying users by Discord mention, Discord ID, or Last.fm username.
```

Comment:
I'm not sure if we should add this since this only applies to a few commands.

Reply:
I know it only applies to a few commands, is there a way we can specify that where identifying a user is supported, it always supports those parameters? it looks like we have standard text for other commands as you mentioned in your other comment, should we just update that standard text to include `Discord mention, Discord ID, or Last.fm username`? that info is extremely useful as a power user who doesn't want to ping other Discord users all the time but I had to ask in the .fmbot Discord before I found out this was the case